R/rdeephaven/R/aggregate_wrapper.R

```diff
@@ -14,80 +14,135 @@ Aggregation <- R6Class("Aggregation",
 
 ### All of the functions below return an instance of the above class
 
+#' @description
+#' Aggregation function that computes the first value of each column in `cols`, for each aggregation group.
+#' Intended to be used in a call to `TableHandle$agg_by()`.
+#' @param cols String or list of strings denoting the names of the columns to aggregate.
+#' @return Result of the aggregation.
```

Comment:
This documentation is inaccurate and should largely be ripped off from the python equivalent.  The python equivalent is `Creates a First aggregation`, which indicates that the method is creating an aggregation.  What is here seems to indicate that the function actually performs an aggreation, which is inaccurate.

I will review the rest of the file after changes are made.